DNN Platform/Library/Entities/Tabs/TabController.cs

```diff
@@ -1002,6 +1002,16 @@ public void ClearCache(int portalId)
             CacheController.FlushPageIndexFromCache();
         }
 
+        public void RefreshCache(int portalId, int tabId)
+        {
+            var portalTabs = GetTabsByPortal(portalId);
+            if (portalTabs.WithTabId(tabId) != null)
+            {
+                var updateTab = GetTab(tabId, portalId, true);
+                portalTabs.RefreshCache(updateTab);

```

Comment:
I'm not convinced about `RefreshCache` of `TabCollection`.

I think I would rename it to be only removal and add the tab again here, inside `TabController`

Reply:
Hi @tpluscode , TabCollection have several internal list and dictionaries, it can't access outside, so we have to provide this method in TabCollection.